packages/terra-toggle-button/tests/wdio/toggle-button-spec.js

```diff
@@ -0,0 +1,195 @@
+/* global before, browser, Terra */
+
+describe('Toggle Button', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-toggle-button/toggle-button/default-toggle-button'));
+
+    // For consistency with test below
+    it('should show a closed button', () => {
+      const screenshot = browser.checkElement('.button');
+      expect(screenshot).to.matchReference('withinTolerance');
```

Comment:
Lets update these to use `Terra.should.matchScreenshot()`

you can select `.button`  like
```
Terra.should.matchScreenshot({ selector: '.button' })`
```

Reply:
We will want to indicate when the screenshot is open or closed in the naming, so we will want to pass `open` and `closed` to `Terra.should.matchScreenshot()` as well. Take a look at [terra-toggle's](https://github.com/cerner/terra-core/blob/master/packages/terra-toggle/tests/wdio/toggle-spec.js) / [terra-overlay's](https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/tests/wdio/overlay-spec.js) screenshots for examples.